lib/Doctrine/ORM/Query/FilterCollection.php

```diff
@@ -194,6 +192,9 @@ public function getHash()
             $filterHash .= $name . $filter;
         }
 
+        $this->filterHash = md5($filterHash);
```

Comment:
This change looks dangerous. The first call to the method will return the raw `$filterHash` while subsequent calls will return the MD5 sum. Why do you MD5 that value anyway?